src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensSchema.cs

```diff
@@ -116,7 +116,10 @@ public SemanticTokensSchema(IReadOnlyDictionary<string, string> tokenTypeMap)
         }
 
         public static SemanticTokensSchema GetSchema(ClientCapabilities capabilities)
-            => capabilities.HasVisualStudioLspCapability()
+            => GetSchema(capabilities.HasVisualStudioLspCapability());
```

Comment:
Let me know if you want me to remove this overload completely. Every caller could just pass `capabilities.HasVisualStudioLspCapability()` and it would all work the same.

Reply:
I think its reasonable to remove the client capabilities overload entirely